front_end/src/app/(main)/questions/track-record/components/charts/scatter_plot.tsx

```diff
@@ -224,7 +224,7 @@ const ScatterPlot: React.FC<HistogramProps> = ({
         {hoverData ? (
           <>
             <Link
-              href={`/questions/${hoverData.question_id}`}
+              href={`/questions/${hoverData.post_id}`}
```

Comment:
I can see why this flavour of bug keeps on happening; this url structure <-> data model entity relationship is very confusing.